source/stylesheets/components/_button.scss

```diff
@@ -0,0 +1,14 @@
+@mixin search-button {
```

Comment:
This should be moved to _mixins.scss. Why this is mixin? It doesn't have any arguments. It should be simple class then